routes/bull/bull.ts

```diff
@@ -15,4 +15,21 @@ router.post("", async (req: Request, res: Response) => {
     res.json(new BullResponse(bullFromDb));
 })
 
+router.delete("/delete", async (req: Request, res: Response) => {
+    try{
+        const bullId: number = req.body.bullId;
```

Comment:
zamiast przesyłać id byka w body powinnaś wziąć to jako parametr z linku: /delete/:id